### PR TITLE
Fix ethereumfaucet.info

### PR DESCRIPTION
**Pull request by ghost**
https://github.com/AdguardTeam/AdguardFilters/issues/101665

### CODE REVIEW

File: AnnoyancesFilter/sections/tweaks.txt [hunk 1]
```diff
@@ -289,6 +289,8 @@ newsweek.com##iframe[src="http://sns.europe.newsweek.com/newsletter?popup=1"]
 !######### JS tweaks ##########
 !##############################
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/101665
+ethereumfaucet.info#%#//scriptlet("set-constant", "tr", "1")
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/tweaks.txt:293]: It seems this rule breaks the site.

Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/tweaks.txt:293]: This rule works, please change your rule to this. 
`ethereumfaucet.info#%#//scriptlet("adjust-setInterval", "ct", "1000", "0.02")`

File: EnglishFilter/sections/content_blocker.txt [hunk 2]
```diff
@@ -1597,6 +1597,8 @@ maclife.de,kuechengoetter.de,prad.de,infranken.de,nachrichten.at,np-coburg.de,fr
 ! https://github.com/AdguardTeam/AdguardFilters/issues/47876
 topbusiness24.ru,mydomsam.ru,hubuhu.ru,moysadinfo.ru,mybusinessplus.ru,dukand.ru,mydizayn.ru,polzadom.ru,umnodachnik.ru,delnyesovety.ru,soverhenie.ru,mirinteresa.ru,uskorit.ru,dskyar.ru,biscand.ru,smartid24.ru,stroysaminfo.ru,topsovety.ru,sotsvetiya.ru,imdaily.ru,turizmnaok.ru,zdorovyobrazzhizni.ru,kinomann.ru,techinfolife.ru,dommoysad.ru,rteinfo.ru,psisovety.ru,kvilit.ru,speshit.ru,ollss.ru#@#.ad-placement
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/101665
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=ethereumfaucet.info
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/content_blocker.txt:1601]: please remove the rules for antiadblock

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/content_blocker.txt (hunk 2 above)
Comment (piquark6046) [reply EnglishFilter/sections/content_blocker.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/101689/commits/9a0524d3d7648f6df4595fa3614f5f0017cd2d7c

File: AnnoyancesFilter/sections/tweaks.txt (hunk 1 above)
Comment (piquark6046) [reply AnnoyancesFilter/sections/tweaks.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/101689/commits/07c2400c89a97bb9751789e5102200d777d9eb0c

Comment (piquark6046) [reply AnnoyancesFilter/sections/tweaks.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/101689/commits/07c2400c89a97bb9751789e5102200d777d9eb0c

File: EnglishFilter/sections/antiadblock.txt [hunk 3]
```diff
@@ -28,7 +28,7 @@
...
...,sqlserveregitimleri.com,tokuvn.com,texture-packs.com,bestwego.xyz,taregna.com,pcwarehub.com,phpscripttr.com,moddedguru.com,oyuncusoruyor.com,pbarecap.ph,sourds.net,teknobalta.com,tinyppt.com,tvinternetowa.info,nullpress.net,nulljungle.com,udemydownload.com,doffeecafe.me,5.182.211.129,latesthdmovies.digital,souqdesign.ru,short.ctvb1.info,hataphim.com,omerbozdi.com.tr,gigacourse.com,codingtuting.com,linkbuk.in,movieslegacy.com,gaypornhdfree.*,javgayplus.com,netfuck.net,javsex.to,pelismart.com,htstreaming.com,themehits.com,freewebcart.com,shineads.in,watchmdh.to,temi.pl,solidfile.net,downloadtutorials.net,win10.vn,nullcode.cc,movieshub.top,share.wogan.info,legaartis.pl,freetraderdownload.com.br,masihbelajar.com,legendary-rt-textures.xyz,blogging-techies.com,privatemoviez.com,jpshowbiz.us,courseclub.me,behindzscene.net,magicgameworld.com,conhecimentolivre.net,latestmovie4u.com,eductin.com,rtilinks.com,blisseyhusband.in,cronachedellacampania.it,applexgen.com,nulleb.com,desktophut.com,moviedhun.xyz,grafixfather.com,pinsystem.co.uk,mastiflix.com,streamcr7.com,globalrph.com,cyberdefensemagazine.com,vectogravic.com#%#//scriptlet("prevent-addEventListener", "DOMContentLoaded", "adsBlocked")
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:31]: I see it in the code, but do you need this rule?

Comment (Sergey-Lyapin) [reply EnglishFilter/sections/antiadblock.txt:31]: @piquark6046 
This rule is redundant, please remove the domain.

**Review by zloyden — APPROVED** [review]

File: EnglishFilter/sections/antiadblock.txt (hunk 3 above)
Comment (zloyden) [reply EnglishFilter/sections/antiadblock.txt]: Well, I also don't see the need for this rule, removed it from the pr.

**Pull request merged**